tsconfig.json

```diff
@@ -18,7 +18,9 @@
       "@lingui/cli/api": ["./packages/cli/src/api"],
       "@lingui/react": ["./packages/react/src"],
       "@lingui/conf": ["./packages/conf/src"],
-      "@lingui/macro": ["./packages/macro/src"]
+      "@lingui/macro": ["./packages/macro/src"],
+      "@lingui/format-po": ["./packages/format-po/src/po.ts"],
```

Comment:
`po-gettext` is missing?

Reply:
Intentionally. It's not used anywhere in the other codebase. csv also missing